### PR TITLE
Use `gts` instead of `gulp`

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  "author": {
+    "name": "Google Inc."
+  },
   "description": "Stackdriver Debug Agent for Node.js",
   "main": "./build/src/index.js",
   "types": "./build/types/index.d.ts",
-  "repository": "googlecloudplatform/cloud-debug-nodejs",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/googlecloudplatform/cloud-debug-nodejs.git"
```
Comment (ofrobots) [inline package.json]: keep the original style. Is `gts` still making these changes? It seems we need to a new release to pick up https://github.com/google/ts-style/pull/55.

File: package.json [hunk 2]
```diff
@@ -67,18 +65,72 @@
...
   "files": [
     "CHANGELOG.md",
     "LICENSE",
     "README.md",
     "build/src",
     "build/types"
-  ]
-}
+  ],
+  "contributors": [
```
Comment (ofrobots) [inline package.json]: drop this along with `readmeFilename`, bugs, and homepage keys.

File: tsconfig.json [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
+    "src/**/*.ts",
+    "test/*.ts",
+    "test/**/*.ts",
+    "system-test/*.ts",
+    "system-test/**/*.ts"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
+}
```
Comment (ofrobots) [inline tsconfig.json]: add new line.

File: package.json [hunk 4]
```diff
@@ -67,18 +65,72 @@
...
+      "name": "Kelvin Jin",
+      "email": "kelvinjin@google.com"
+    }
+  ],
+  "readmeFilename": "README.md",
+  "bugs": {
+    "url": "https://github.com/googlecloudplatform/cloud-debug-nodejs/issues"
+  },
+  "homepage": "https://github.com/googlecloudplatform/cloud-debug-nodejs#readme"
+}
```
Comment (ofrobots) [inline package.json]: add newline

File: package.json (hunk 1 above)
Comment (ofrobots) [reply package.json]: gts 0.4.0 published. It will no longer pollute the `package.json`.

**Comment by DominicKramer** [comment]
Closing in favor of #344.

**Pull request closed without merging**